pkg/nfd-master/nfd-master.go

```diff
@@ -294,6 +299,12 @@ func (m *nfdMaster) prune() error {
 			return fmt.Errorf("failed to prune labels from node %q: %v", node.Name, err)
 		}
 
+		// Prune taints
+		err = m.setTaints(c, cli, crt, node.Name)
```

Comment:
No need to declare any variables above

```suggestion
		err = m.setTaints(context.TODO(), cli, []corev1.Taint{}, node.Name)
```